Directory.Build.props

```diff
@@ -9,6 +9,11 @@
     <Import Project="build\config.props" />
     <Import Project="build\common.project.props" />
   </ImportGroup>
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
Does this make `git format` in CI redundant?  If so, we should remove it from the pipeline: https://github.com/NuGet/NuGet.Client/blob/bd0764b4a5b78eea77c61f5b82078623c4fc0cfe/eng/pipelines/templates/Static_Source_Analysis.yml#L77-L81